### ISSUE TITLE
The list doesn't continue after 3 loads

### ISSUE THREAD

**Issue by Auditormadness9** [hoothin/UserScripts#277]
Hello, might you know why when I scroll through this list it only loads 2 times after which it stops and refuses to continue? Here is my custom rule written for it. Thanks in advance!

```   {
        "name": "Garry's Mod",
        "url": "^https?://steamcommunity\\.com/",
        "pageElement": "body > div.responsive_page_frame.with_header > div.responsive_page_content > div#responsive_page_template_content.responsive_page_template_content > div#ig_bottom.default > div#mainContents > div#profileBlock.clearfix > div",
        "action": 2
    },
```

And this is the webpage: https://steamcommunity.com/workshop/browse/?appid=4000&searchtext=PM+%7C+NPC+%7C+playermodel&childpublishedfileid=0&browsesort=mostrecent&section=readytouseitems&requiredtags%5B0%5D=Model&created_date_range_filter_start=0&created_date_range_filter_end=0&updated_date_range_filter_start=0&updated_date_range_filter_end=0&actualsort=mostrecent&p=1

**Comment by hoothin** [comment]
Tks for report, it's fixed in 1.9.33.31.

**Comment by Auditormadness9** [comment]
that was quick, thanks!
